source/includes/deployment-region-auth-callback-urls.rst

```diff
@@ -8,7 +8,7 @@
    * - | **Global**
      - .. code-block:: text
 
-          https://stitch.mongodb.com/api/client/v2.0/auth/callback
+          https://realm.mongodb.com/api/client/v2.0/auth/callback 
```

Comment:
IIRC this didn't work when we initially moved from the Stitch -> Realm brand. Did you confirm that this URL works now?

Also, do we need to update the local URLs in the other rows of this table? They all include "stitch" too.

Reply:
@nlarew we tested out both URLs and didn't have any issues having either the stitch one or the realm one as the redirect URI, which initially seemed a bit strange to me. However, Lindsey also confirmed with product and they verified realm.mongodb.com as the correct one. See: https://mongodb.slack.com/archives/C41SX3C4D/p1665162980386979